test/e2e/case20_informonly_test.go

```diff
@@ -0,0 +1,97 @@
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = Describe("Test 'InformOnly' ConfigurationPolicies", Ordered, func() {
+	const (
+		case20PolicyName    string = "case20-policy-informonly"
+		case20PolicyYaml    string = "../resources/case20_policy_informonly/case20-parent-policy.yaml"
+		case20ConfigMapName string = "test-config"
+		case20ConfigMapYaml string = "../resources/case20_policy_informonly/case20-configmap.yaml"
+		case20ConfigPlcName string = "create-configmap"
+	)
+
+	BeforeEach(func() {
+		hubApplyPolicy(case20PolicyName, case20PolicyYaml)
+	})
+
+	AfterEach(func() {
+		By("Deleting a policy on the hub in ns:" + clusterNamespaceOnHub)
+		_, err := kubectlHub("delete", "-f", case20PolicyYaml, "-n", clusterNamespaceOnHub)
+		var e *exec.ExitError
+		if !errors.As(err, &e) {
+			Expect(err).ShouldNot(HaveOccurred())
+		}
+		opt := metav1.ListOptions{}
+		utils.ListWithTimeout(clientManagedDynamic, gvrPolicy, opt, 0, true, defaultTimeoutSeconds)
+	})
+
+	AfterAll(func() {
+		_, err := kubectlManaged("delete", "configmap", case20ConfigMapName, "-n",
+			clusterNamespace, "--ignore-not-found")
+		var e *exec.ExitError
+		if !errors.As(err, &e) {
+			Expect(err).ShouldNot(HaveOccurred())
+		}
+	})
+
+	Describe("Override remediationAction in spec", func() {
+		Context("When the CR does not exist", func() {
+			It("Should be noncompliant", func() {
+				Eventually(func() interface{} {
+					plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigurationPolicy,
+						case20ConfigPlcName, clusterNamespace, true, defaultTimeoutSeconds)
+
+					return utils.GetComplianceState(plc)
+				}, defaultTimeoutSeconds, 1).Should(Equal("NonCompliant"))
+			})
+		})
+
+		Context("When the CR exists", func() {
+			BeforeAll(func() {
+				_, err := kubectlManaged("apply", "-f", case20ConfigMapYaml, "-n", clusterNamespace)
+				var e *exec.ExitError
+				if !errors.As(err, &e) {
+					Expect(err).ShouldNot(HaveOccurred())
+				}
+			})
+
+			It("Verifies creation of configmap resource", func() {
+				cfmap, _ := clientManaged.CoreV1().ConfigMaps(clusterNamespace).Get(context.TODO(),
+					case20ConfigMapName, metav1.GetOptions{})
+
+				Expect(cfmap).NotTo(BeNil())
+			})
+
+			It("Should be compliant", func() {
+				Eventually(func() interface{} {
+					plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigurationPolicy,
+						case20ConfigPlcName, clusterNamespace, true, defaultTimeoutSeconds)
+
+					return utils.GetComplianceState(plc)
+				}, defaultTimeoutSeconds, 1).Should(Equal("Compliant"))
+			})
+
+			It("Should have remediationAction=inform", func() {
+				Eventually(func() interface{} {
+					plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigurationPolicy,
+						case20ConfigPlcName, clusterNamespace, true, defaultTimeoutSeconds)
+
+					return plc.Object["spec"].(map[string]interface{})["remediationAction"]
+				}, defaultTimeoutSeconds, 1).Should(Equal("inform"))
+			})
```

Comment:
So I think this is the "real" test we're interested in. I think we should also have another test for when the Policy doesn't have a "top-level" `remediationAction` set - what happens then?